src/main/java/com/sap/sgs/phosphor/fosstars/data/github/HasSecurityTeam.java

```diff
@@ -40,8 +40,8 @@ protected Feature supportedFeature() {
   protected Value fetchValueFor(GitHubProject project) throws IOException {
     logger.info("Figuring out if the project has a security team ...");
     if (securityTeam.existsFor(project.url())) {
-      HAS_SECURITY_TEAM.value(true);
```

Comment:
Yup, that was me

https://github.com/SAP/fosstars-rating-core/commit/e9116146053d320afd5a62a7895c0e6f89a3bf57#diff-a021b7a60f2d27e71c1a72712ac83344

Reply:
Not thoroughly reviewed also.